glommio/src/parking.rs

```diff
@@ -229,6 +230,52 @@ impl SharedChannels {
     }
 }
 
+/// Allows information about the current state of this reactor to be consumed
+/// by applications.
+#[derive(Default, Debug, Copy, Clone)]
+pub struct IoStats {
+    files_opened: u64,
+    files_closed: u64,
+    files_reads: u64,
```

Comment:
would be nice to account Direct I/O vs Buffered I/O separately.
It seems to me this could be useful to discover performance bugs where you think you opened the file in direct I/O mode but ends up buffered because of some fallback.

Reply:
The new version now tracks buffered and unbuffered IO separately